<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQQ3ClientClass</class>
 <widget class="QMainWindow" name="MyQQ3ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQQ3Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>0</y>
       <width>95</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ChatButton">
        <property name="text">
         <string>聊天</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ContactButton">
        <property name="text">
         <string>通讯录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CollectButton">
        <property name="text">
         <string>收藏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FileButton">
        <property name="text">
         <string>文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="mainStackedWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>751</width>
       <height>471</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>0</y>
         <width>471</width>
         <height>471</height>
        </rect>
       </property>
       <widget class="QPushButton" name="sendMessagePushButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>350</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>发消息</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>411</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="nickNameTextDisplay">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="quoteTextDisplay">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="profileDisplay">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>备注：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>地区：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>MyQQ3号：</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="markNameLineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="cityTextDisplay">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MyQQ3NameDisplay"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="add">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="commit">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>commit</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_1">
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-9</y>
         <width>211</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="ChatRecordInputPlainTextEdit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>350</y>
         <width>531</width>
         <height>131</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QListView" name="ChatHistoryListView">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>1</y>
         <width>531</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
     <widget class="QWidget" name="page_5"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
